dev-requirements.txt

```diff
@@ -219,6 +221,10 @@ darker==1.4.1 \
     --hash=sha256:c63cbfcf707987bc5cb3da30eaf5f2348aabc03f96b7b69bc31a34e96d8f2b8a \
     --hash=sha256:d423f4d791ff9a752a4a567a074d6c352cf6eb938d24d026a5799fcaa739c6c5
     # via -r dev-requirements.in
+distro==1.7.0 \
+    --hash=sha256:151aeccf60c216402932b52e40ee477a939f8d58898927378a02abbe852c1c39 \
```

Comment:
WHy the extra hashes here?

Reply:
what do you mean by extra? :thinking: these are the hashes for `distro`

Reply:
oh, I see